packages/patterns-react/src/patterns/SimpleLongForm/CardLink.js

```diff
@@ -0,0 +1,72 @@
+import React, { useLayoutEffect, useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import { settings } from 'carbon-components';
+import { ArrowRight20 } from '@carbon/icons-react';
+const { prefix } = settings;
+
+/**
+ * Card Link Component
+ *
+ * @param {object} props props object
+ * @param {string} props.href card link link
+ * @param {string} props.text card link text
+ * @param {string} props.target card link target
+ * @returns {*} CardLink component
+ */
+const CardLink = ({ link: { href, text, target } }) => {
+  const [height, setHeight] = useState(0);
+  const targetRef = useRef();
```

Comment:
Can this be swapped for the abstracted `CardLink` component?
cc: @photodow

Reply:
That's the plan I believe. I know there are some specs still coming from Wonil, so hopefully we can align more when that comes, but once #554 gets merged we should be able to replace child component.

Reply:
Yeah, We can swap CardLink component, we should update and replace child component soon once specs for CardLink component is available.